CircleciScripts/setup_private_specs.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This script generates a private spec repo in your $HOME/.aws-amplify
+# directory, to enable `pod lib lint`. It also makes it easier to do
+# development against unreleased versions of Amplify, at least until we start
+# releasing nightly builds.
+
+set -e
+
+declare -r LOCAL_SPEC_GIT_ROOT="$HOME/.aws-amplify/amplify-ios/amplify-ios-podspecs.git"
+declare -r LOCAL_SPEC_REPO_NAME="amplify-ios-local-specs"
+declare -r COCOAPODS_REPO_DIR="${HOME}/.cocoapods/repos/${LOCAL_SPEC_REPO_NAME}"
+
+function init_spec_git_root {
+  # If directory exists, assume it's correctly set up; don't attempt any
+  # repairs or validation
+  if [[ -d "${LOCAL_SPEC_GIT_ROOT}" ]] ; then
+    return 0
+  fi
+
+  mkdir -p "${LOCAL_SPEC_GIT_ROOT}"
+  old_pwd="${PWD}"
+  cd "${LOCAL_SPEC_GIT_ROOT}"
+  git init --bare
+  cd "${old_pwd}"
```

Comment:
Not required, but i prefer using pushd/popd.  For example this could be:
```
mkdir -p ${LOCAL_SPEC_GIT_ROOT}
pushd ${LOCAL_SPEC_GIT_ROOT}
git init --bare
retval=$?
popd
return ${retval}
```

Reply:
Thanks, I'll update in the next rev, but after RC release, just to make sure I don't have to go through another round of testing